src/components/ImgEditor/types.ts

```diff
@@ -13,4 +17,8 @@ export type Props<T extends FieldValues, K extends keyof T> = {
   accept: string[];
   classes?: Classes;
   aspect: [number, number];
+  /**
+   * Maximum image size in bytes
```

Comment:
Adding JSDoc comments can be very helpful during development as it provided useful info in code editors (see image for on-hover info in VS Code).
![image](https://user-images.githubusercontent.com/19427053/227508010-496ea113-c0fa-4ff3-a4c1-2a390b0c1e1b.png)

We should make it a habit to add them, at least for fields/params whose purpose is not immediately obvious.